Projects/18_Clustering/Predict_Sales.py

```diff
@@ -4,12 +4,11 @@
 df_offers.columns = ["offer_id", "campaign", "varietal", "min_qty", "discount", "origin", "past_peak"]
 
 # import data for transactions
-df_transactions = pd.read_excel("./WineKMC.xlsx", sheetname=1)
-df_transactions.columns = ["customer_name", "offer_id"]
+df_transactions = pd.read_excel("./WineKMC.xlsx", sheetname=1,names=["customer_name", "offer_id"])
```

Comment:
Add a space after commas. Here's a good code [style guide](http://docs.python-guide.org/en/latest/writing/style/) for Python. Also consider using [PyCharm](https://www.jetbrains.com/pycharm/download/) as your IDE for non-Jupyter work.